manager/manager.go

```diff
@@ -688,12 +688,54 @@ func (m *manager) createContainer(containerName string) error {
 		}
 	}
 
+	subcontainers := m.getSubcontainerContainerData(containerName)
+	cont.contSubcontainers = subcontainers
+	m.addAsSubcontainer(cont, namespacedName)
+
+	path, err := cont.handler.GetCgroupPath("cpu")
+	if err != nil {
+		glog.Errorf("could not get cgroup path for container %v. got error %v", cont.info.Name, err)
+	} else {
+		cont.cgroupPath = path
+	}
+
 	// Start the container's housekeeping.
 	cont.Start()
 
 	return nil
 }
 
+func (m *manager) getSubcontainerContainerData(containerName string) map[namespacedContainerName]*containerData {
+	subcontainers := make(map[namespacedContainerName]*containerData)
+	m.containersLock.RLock()
+	defer m.containersLock.RUnlock()
+	for namespacedNameKey, containerDataValue := range m.containers {
+		if strings.HasPrefix(namespacedNameKey.Name, containerName) && namespacedNameKey.Name != containerName {
```

Comment:
Does this add all containers as subcontainers of "/". Do we track the immediate children anywhere? We are only interested in immediate children.